### DISCUSSION TITLE
I can't seem to be able to export my data to Excel or CSV

### DISCUSSION THREAD

**Discussion by ZephyrDogma** [asreview/asreview#1957] [Q&A]
[nightfiredetection (1).zip](https://github.com/user-attachments/files/18028626/nightfiredetection.1.zip)
Im trying as the title implies to export my labeled dataset to Excel or CSV and i get this error:

ERROR:root:Failed to export the xlsx dataset. [Errno 2] No such file or directory: '/usr/local/lib/python3.8/site-packages/asreview/webapp/project_folder/nightfiredetection/tmpr7dg74q5/export_dataset.xlsx'

For further information i have AsReview installed in docker container

**Comment by J535D165** [comment]
Thanks for reporting. It might have to do this https://github.com/asreview/asreview/releases/tag/v1.6.2. What version of ASReview are you using?
